packages/mongo-nav/src/org-nav/OrgNav.tsx

```diff
@@ -141,15 +149,15 @@ export default function OrgNav({
         />
 
         <ul className={ulContainer}>
-          {paymentVariant && current?.paymentStatus && (
+          {paymentVariant && showPaymentStatus && (
             <li>
               <Badge
                 className={css`
                   margin-right: 25px;
                 `}
                 variant={paymentVariant}
               >
-                {current.paymentStatus.toUpperCase()}
+                {current?.paymentStatus?.toUpperCase()}
```

Comment:
Two things:

First, to make the transition to rollup a bit smoother, can we use the old syntax for now? Not ideal or permanent, but we won't be able to use optional chaining with it for a bit.

Second, what's the case where paymentStatus is truthy, but also isn't a string? The `?.toUpperCase` strikes me as unnecessary.

Reply:
it's odd because we are checking for the presence of both current and current.paymentStatus with `showPaymentStatus` so not sure why I even need to check down here

Reply:
when I inline the check, that goes away... perhaps I should just do that